tests/service/data/TestTP/ProvidedTypes.fs

```diff
@@ -3220,8 +3220,6 @@ module internal AssemblyReader =
           systemRuntimeScopeRef: ILScopeRef }
         override __.ToString() = "<ILGlobals>"
 
-    [<AutoOpen>]
```

Comment:
AutoOpen is the same story - rather then removing it, it makes more sense to extend it also on structs.